webview/src/shared/variables.scss

```diff
@@ -27,14 +27,14 @@ $row-hover-background-color: var(--vscode-list-hoverBackground);
 
 $accent-color: var(--button-primary-background);
 
-$indicator-badge-background: var(--vscode-badge-background);
-$indicator-badge-foreground: var(--vscode-badge-foreground);
-$indicator-badge-border: $row-bg-color;
-$indicator-selected-badge-border: $row-bg-selected-color;
-$indicator-selected-badge-background: $accent-color;
-$indicator-selected-badge-foreground: var(--button-primary-foreground);
-
-$star-switch-selection-fill-color: var(--vscode-list-focusHighlightForeground);
+$indicator-badge-background: var(--vscode-activityBarBadge-background);
+$indicator-badge-foreground: var(--vscode-activityBarBadge-foreground);
+
+$row-action-icon-color: var(--vscode-icon-foreground);
+$row-action-selected-icon-color: var(
```

Comment:
`--vscode-icon-foreground` appears to be the variable used for tree chevrons! 

Also, I realized that the default theme trees don't actually have their chevrons stay the same color when selected. Looks like they actually change their color based on the variable `--vscode-list-activeSelectionIconForeground`. The light theme and a couple other themes use this variable as well:

<img width="312" alt="image" src="https://user-images.githubusercontent.com/43496356/189405887-35ed37e2-dd0d-4aa8-8831-f510a1be675c.png">
<img width="286" alt="image" src="https://user-images.githubusercontent.com/43496356/189405948-93034d5e-be91-4cd6-972b-d92df2b48071.png">

Matching how vscode themes work, I had our table update its chevrons if it's selected with `--vscode-list-activeSelectionIconForeground`, defaulting to the `--vscode-icon-foreground` since not all themes use the active variable.

Reply:
Looks like the text also changes color.

Reply:
Yes! Probably also relies on some kind of `active` variable 🤔. Can look into it and open a pr at a later date!

Reply:
🙏🏻